### PR TITLE
When connecting to wifi network without PSK, key_mgmt need to be sat …

### PR DESCRIPTION
**Pull request by sconrads**
Needed to use the MusicBox on a wifi network without psk. To make it work I had to change the /etc/wpa.conf
Thanks for a great application :)

### CODE REVIEW

File: filechanges/opt/musicbox/startup.sh [hunk 1]
```diff
@@ -91,16 +91,31 @@ chmod u+s /sbin/shutdown
...
+                scan_ssid=1
+            }
+        EOF
+    else
+        cat >/etc/wpa.conf <<EOF
+            ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
+            update_config=1
+            network={
+                ssid="$INI__network__wifi_network"
+                proto=RSN
```
Comment (kingosticks) [inline filechanges/opt/musicbox/startup.sh]: Do you need this?

Comment (sconrads) [inline filechanges/opt/musicbox/startup.sh]: I need this to be able to connect to an open wifi network at work. The wifi security is web signon, not WEP og WPA. 
After booting MusicBox i log on to the network using a curl command.
I guess other users also would like to logon to open network, ie. when you use MAC authentication.

Comment (kingosticks) [inline filechanges/opt/musicbox/startup.sh]: Don't get me wrong, supporting open WiFi networks is something I've been meaning to get around to (hence #286). My comment regarding line 113 is specifically about the need for `proto=RSN`. Looking at the wpa_supplicant manual I wouldn't have thought it was necessary.

Comment (sconrads) [inline filechanges/opt/musicbox/startup.sh]: Sorry, I removed it, and it works fine without it. I will update the  pull request tonight.

**Comment by kingosticks** [comment]
Thank you. Merged in https://github.com/woutervanwijk/Pi-MusicBox/commit/e8d06c281c7fc60fab535b6398efe512065f0f44

**Pull request closed without merging**